ern-api-gen/src/DefaultCodegen.ts

```diff
@@ -1372,7 +1372,9 @@ export default class DefaultCodegen {
   }
 
   public addAdditionPropertiesToCodeGenModel(codegenModel, swaggerModel) {
-    const mapProperty = new MapProperty(swaggerModel.getAdditionalProperties())
+    const swaggerAdditionalProperties = swaggerModel.getAdditionalProperties()
+    const mapProperty = new MapProperty()
+    mapProperty.setAdditionalProperties(swaggerAdditionalProperties)
```

Comment:
👀 --this was key!  as you can see, the `swaggerAdditionalProperties` were previously being passed into the constructor, who just threw them away!  `setAdditionalProperties` is the mechanism provided internally to set these

Reply:
whoa, we forked it huh! kinda. hmm.  ya, not seeing those associated java deps.  i _really like and respect the spirit of this ask_--i think it's on the money 💸 .  my java is a bit rusty, but i can get by. it's a bit of an ask to setup a java env in a 3p repo, and track disconnected paired changes in a different lang.  i'm not against it, but i'd like to learn a bit more before agreeing.

i read thru https://github.com/electrode-io/electrode-native/blob/master/docs/overview/contributing.md . if patching a different codebase _should be_ a req to patching this codebase, perhaps we ought throw a patch in there?  also, what do you think about a `.github/contributing.md` maybe just to link to this one, s.t. on PR github prompts folks to read it?

i'm willing to give this a timeboxed effort.  will start shortly

Reply:
It's not "forked" in the sense of a GitHub repository fork, but as you probably noticed, almost all code here is a direct 1:1 translation from the "upstream" `swagger-codegen` Java code to TypeScript. Thus, any logical issue you find in the `ern-api-gen` code (that was not done by us), likely also exists in `swagger-codegen` (unless it has already been fixed in that repo). Currently there is no process of "synchronizing" the code to `electrode-native`.